seirmo/apps/_simulation_app.py

```diff
@@ -25,6 +25,7 @@ def __init__(self):
         super(_SimulationApp, self).__init__()
 
         self._fig_plot = se.IncidenceNumberPlot()
```

Comment:
Could you call this something more specific to the incidence plot, maybe self._inc_fig or so, and then maybe the other self._comp_fig.

Reply:
Or keep both explicit: inicidence_fig and compartment_fig, whatever you prefer